### ISSUE TITLE
Wordpress plugin

### ISSUE THREAD

**Issue by WheresHJ** [codeforboston/internet_EBS#1]
In order to distribute the IEBS to as many sites as possible, we should create a Wordpress plugin that will allow Wordpress users to easily install and configure the IEBS widget.

**Comment by chrismetcalf** [comment]
A pretty good idea. I'd add Drupal to the list as well.

But I'd save that for V1.5 or V2.0. We should focus first on getting the
core functionality in.

**Comment by WheresHJ** [comment]
Yes, definitely! I had tagged these both as "ideas" to try and imply their v.2 nature, but maybe I was unclear! I renamed it "backlog."
